python/create_client/create_utils.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import platform
+import subprocess
+
+import sgtk
+
+SHOTGUN_CREATE_PATH = dict()
+SHOTGUN_CREATE_PATH_ENV = dict()
+
+SHOTGUN_CREATE_PATH["windows"] = os.path.abspath(os.path.join(
+    os.sep, "ProgramFiles", "Autodesk", "ShotgunCreate", "bin", "ShotgunCreate.exe"))
+SHOTGUN_CREATE_PATH_ENV["windows"] = "SHOTGUN_CREATE_WINDOWS_PATH"
+
+SHOTGUN_CREATE_PATH["darwin"] = os.path.abspath(os.path.join(
+    os.sep, "Applications", "Autodesk", "Shotgun Create.app", "Contents", "MacOS", "Shotgun Create"))
+SHOTGUN_CREATE_PATH_ENV["darwin"] = "SHOTGUN_CREATE_DARWIN_PATH"
+
+SHOTGUN_CREATE_PATH["linux"] = os.path.abspath(os.path.join(
+    os.sep, "opt", "Autodesk", "ShotgunCreate", "bin", "ShotgunCreate"))
+SHOTGUN_CREATE_PATH_ENV["linux"] = "SHOTGUN_CREATE_LINUX_PATH"
+
+PLATFORM = platform.system().lower()
```

Comment:
Or you could use a `sgtk.util.ShotgunPath`, pass in your three values for each platform and then do `path.current_os`.

I assume this is temporary and we'll see a more robust version of Create detection in the future, since a DCC could have been launched not from Create?

Reply:
Done. About your question, it's to be defined.